src/server/common/model/answer/quantity-options/quantity-options.js

```diff
@@ -0,0 +1,27 @@
+import { RadioButtonAnswer } from '../radio-button/radio-button.js'
+
+/** @import {RadioButtonConfig} from '../radio-button/radio-button.js' */
+
+/**
+ * export @typedef {'yes'|'no'|'unknown'} QuantityOptionsData
+ * @typedef {{ quantityOptions: QuantityOptionsData }} QuantityOptionsPayload
+ */
+
+/**
+ * @augments {RadioButtonAnswer<QuantityOptionsPayload>}
+ */
+export class QuantityOptionsAnswer extends RadioButtonAnswer {
+  /** @type {RadioButtonConfig} */
+  static config = {
+    payloadKey: 'quantityOptions',
```

Comment:
I'd rename this key as "movingMoreThan75Animals", because otherwise the raw data won't make sense.

Reply:
Actually, it's the questionKey that really matters, rather than this one.